packages/react/src/components/CardCodeEditor/CardCodeEditor.test.e2e.jsx

```diff
@@ -64,7 +64,7 @@ describe('CardCodeEditor loaded editor test', () => {
 
     // This component throws a network error with too many calls to the cdn script it loads so adding snapshot to existing instance
     onlyOn('headless', () => {
-      cy.findByTestId('ComposedModal').compareSnapshot('CardCodeEditor');
+      cy.findByTestId('ComposedModal').compareSnapshot('CardCodeEditor', 0.2);
```

Comment:
any specific reason ?

Reply:
Just adding a threshold, instead of 0, locally the tests were passing but not here, maybe some  compare not 100% equal, so just added a bit of threshold.